tests/page_test.go

```diff
@@ -115,13 +115,13 @@ func TestPageEvaluate(t *testing.T) {
 			{
 				"promise",
 				`async () => { return await new Promise((res, rej) => { rej('rejected'); }); }`,
-				`exception "Uncaught (in promise)" (0:0): rejected`,
+				"evaluating JS: rejected",
```

Comment:
```suggestion
				"promise rejected",
```

Reply:
Makes sense, but I'll leave it for the new issue.

Reply:
Actually, we should probably keep "evaluating JS", since that's what's happening, and removing it wouldn't make sense for the below cases (`SyntaxError`, `ReferenceError`, etc.). But adding "promise" would be clearer.